db/migrate/20170815175201_swap_column_names.rb

```diff
@@ -0,0 +1,7 @@
+class SwapColumnNames < ActiveRecord::Migration
+  def change
+  	rename_column :scheduled_items, :admission, :admission_temp
+  	rename_column :scheduled_items, :audience, :admission
+  	rename_column :scheduled_items, :admission_temp, :audience
+  end
```

Comment:
Yes, I used this [stackoverflow](https://stackoverflow.com/questions/1992019/how-can-i-rename-a-database-column-in-a-ruby-on-rails-migration) as a reference.

I just deployed it to staging now after finishing the integration test and checked existing schedule slides before and after the deploy and all slides seem to be correct with no data lost.